src/main/java/com/github/manolo8/darkbot/backpage/NovaManager.java

```diff
@@ -0,0 +1,163 @@
+package com.github.manolo8.darkbot.backpage;
+
+import com.github.manolo8.darkbot.backpage.nova.Agent;
+import com.github.manolo8.darkbot.backpage.nova.NovaData;
+import com.github.manolo8.darkbot.backpage.nova.Perk;
+import com.github.manolo8.darkbot.utils.http.Method;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+
+import java.io.IOException;
+
+public class NovaManager {
+    private final BackpageManager backpageManager;
+    private final NovaData data;
+    private final Gson g;
```

Comment:
avoid one-letter names in fields, they're fine in lambdas or small-scoped things, but i'd use the more descriptive ` `gson` here